### ISSUE TITLE
Customize icon color with add button

### ISSUE THREAD

**Issue by JperF** [stripe/stripe-react-native#1022]
**Is your feature request related to a problem? Please describe.**
No issue.

**Describe the solution you'd like**
I want this icon's color to be customizable.
<img width="423" alt="Screen Shot 2022-06-28 at 2 00 16 PM" src="https://user-images.githubusercontent.com/4474118/176288957-ca05a48b-1c53-4fd3-bb3f-3221fe5ad00f.png">

**Describe alternatives you've considered**
None, `two-shoes#3046` said this was not possible when asking in the official stripe discord. 

**Additional context**
N/A

**Comment by JperF** [comment]
Does the `style: 'alwaysLight'` enforcement work with android @charliecruzan-stripe ?

**Comment by charliecruzan-stripe** [comment]
no, it's iOS only right now (tracking that in https://github.com/stripe/stripe-react-native/issues/285)

**Comment by wooj-stripe** [comment]
@JperF  - I believe this has been fixed in the underlying Android SDK. The color of the '+' icon should be changed based on the color of component color set through the appearance API.  I'll be optimistically closing this ticket, but feel free to re-open this ticket if this is still an issue in the latest version of the RN SDK.
